wcivf/apps/elections/management/commands/import_elections.py

```diff
@@ -16,24 +16,28 @@ def handle(self, **options):
         for page in pages:
             self.add_elections(page)
 
-        # Now import any metadata from EE
-        self.import_metadata()
+        # Now import any other bits from EE that aren't in YNR
+        self.import_extras()
 
     def add_elections(self, results):
         for election in results["results"]:
             election_obj, created = Election.objects.update_or_create_from_ynr(election)
             if created:
                 self.stdout.write("Added new election: {0}".format(election["name"]))
 
-    def import_metadata(self):
+    def import_extras(self):
         pages = self.get_paginator(
-            "{}/api/elections/?current=true&metadata=1".format(settings.EE_BASE)
+            "{}/api/elections/?current=true".format(settings.EE_BASE)
```

Comment:
Like various other things in this codebase, this will only work properly for `current` elections, so we break historic elections when we rebuild, but that's not a new problem.